_posts/0000-01-02-hpcgirl88.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
```

Comment:
this is **bolb** and here is [link to git](https://guides.github.com/features/mastering-markdown/)